src/Composer/Installers/Installer.php

```diff
@@ -103,6 +105,30 @@ class Installer extends LibraryInstaller
         'prestashop'   => 'PrestashopInstaller'
     );
 
+    /**
+     * Installer constructor.
+     *
+     * Disables installers specified in main composer extra installer-disable
+     * list
+     *
+     * @param IOInterface                              $io
+     * @param \Composer\Composer                       $composer
+     * @param string                                   $type
+     * @param \Composer\Util\Filesystem|null           $filesystem
+     * @param \Composer\Installer\BinaryInstaller|null $binaryInstaller
+     */
+    public function __construct(
+        \Composer\IO\IOInterface $io,
```

Comment:
Since these are already imported, can you just use their short classname? Same for docblocks please.

Reply:
Done